scripts/pipeline.sh

```diff
@@ -13,7 +13,7 @@ TOOLCHAIN_DIR=${1:-$(pwd)/toolchain}
 TTY=${2:-/dev/null}
 
 # Timeout for each step.
-TIMEOUT=90
+TIMEOUT=300
```

Comment:
We should change in ci.py and Makefile as well.

Reply:
Done!